examples/platform_ios_headless.rs

```diff
@@ -0,0 +1,77 @@
+//! Defining an engine in Wasmer is one of the fundamental steps.
+//!
+//! This example builds on that of 'engine_headless.rs' but instead of
+//! serializing a module and then deserializing it again for your host machines target,
+//! We instead create an engine for our target architecture (In this case an ARM64 iOS device),
+//! serialize a simple module to a .dylib file that can be copied to an iOS project and
+//! deserialized/ran using the 'Headless C-API'.
+//!
+//! ```shell
+//! cargo run --example platform-headless-ios --release --features "cranelift"
+//! ```
+//!
+//! Ready?
+
+use std::fs::File;
+use std::path::Path;
+use std::str::FromStr;
+use tempfile::NamedTempFile;
+use wasmer::imports;
+use wasmer::wat2wasm;
+use wasmer::Instance;
+use wasmer::Module;
+use wasmer::RuntimeError;
+use wasmer::Store;
+use wasmer::Value;
```

Comment:
Can we do all this imports in the same line?

Reply:
Updated.